code-of-conduct.html

```diff
@@ -40,8 +40,8 @@
 
       <nav id="navbar" class="navbar order-last order-lg-0">
         <ul>
-          <li><a class="nav-link scrollto" href="/index.html">Home</a></li>
-          <li><a class="nav-link scrollto active" href="/code-of-conduct.html">Code of Conduct</a></li>
+          <li><a class="nav-link scrollto" href="index.html">Home</a></li>
```

Comment:
@basiima what is the essence of this change? why are we changing from `href="/index.html"` to `href="index.html"`

Reply:
"/index.html" is an invalid reference but rather "index.html"

Reply:
> "/index.html" is an invalid reference but rather "index.html"

Could you please share some references that support this?

Reply:
In my view, the the `slash` makes the URL absolute thus telling browsers to look in the current folder for the directory and then the subsequent directories and the file.

References
https://stackoverflow.com/questions/7613274/why-would-a-developer-place-a-forward-slash-at-the-start-of-each-relative-path
https://www.htmlvalidator.com/CSS-HTML-Forum/viewtopic.php?t=2684#:~:text=Paths%20in%20HTML%20(like%20those,to%20access%20a%20local%20file.

Reply:
Oh I didn't know that.  It just didn't seem to route properly on my local development environment.

Reply:
no worries, alright

Reply:
with that adjustment we shall be good to merge this pr